### PR TITLE
Basic UI components

### PR DESCRIPTION
**Pull request by vnshgrg**
Preview:

https://github.com/ue-sho/crypto-tax-simulator/assets/3870529/4eb711ea-f0b6-45ec-8d7a-5cd681c143c1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,10 @@
     "prettier:fix": "prettier --w ."
   },
   "dependencies": {
+    "@radix-ui/react-accordion": "^1.1.2",
+    "@radix-ui/react-dialog": "^1.0.5",
+    "@radix-ui/react-label": "^2.0.2",
+    "@radix-ui/react-popover": "^1.0.7",
```
Comment (vnshgrg) [inline package.json:18]: @ue-sho You should run `npm i` once you pull these changes to your local.

File: src/app/page.tsx [hunk 2]
```diff
@@ -1,113 +1,208 @@
...
-          </h2>
-          <p className="m-0 max-w-[30ch] text-sm opacity-50">
-            Learn about Next.js in an interactive course with&nbsp;quizzes!
-          </p>
-        </a>
+const SectionOne = () => {
+  return (
+    <div className="space-y-4">
+      <Question title="給与収入" help="あなたの職業を選択してください。">
+        <Input type="text" placeholder="給与収入" />
```
Comment (ue-sho) [inline src/app/page.tsx:82]: I think all input types should be `number`.

File: src/app/page.tsx [hunk 3]
```diff
@@ -1,113 +1,208 @@
...
-          className="relative dark:drop-shadow-[0_0_0.3rem_#ffffff70] dark:invert"
-          src="/next.svg"
-          alt="Next.js Logo"
-          width={180}
-          height={37}
-          priority
+      <Card className="w-full max-w-2xl">
+        <Header />
+        <CardContent className="border-y border-slate-200 bg-slate-50/40 pt-6">
+          <Accordion type="single" collapsible className="w-full">
```
Comment (ue-sho) [inline src/app/page.tsx:36]: we should do `<Accordion type="multiple" className="w-full">`.

**Review by ue-sho — COMMENTED** [review]
Thanks for the fast implementation.

File: src/app/page.tsx (hunk 2 above)
Comment (vnshgrg) [reply src/app/page.tsx]: Thanks, addressed here https://github.com/ue-sho/crypto-tax-simulator/pull/1/commits/f3c234563308307188ec4fdb30191fd84d9538ea.

File: src/app/page.tsx (hunk 3 above)
Comment (vnshgrg) [reply src/app/page.tsx]: Indeed, addressed here https://github.com/ue-sho/crypto-tax-simulator/pull/1/commits/d8c2cb2870fe9419ee26312f3da5db28192269db. Thanks :pepe-salute:

**Review by ue-sho — APPROVED** [review]

**Pull request merged**